packages/yourloops/test/integration/assert/stats.ts

```diff
@@ -40,6 +40,18 @@ export const checkStatTooltip = async (patientStatistics: BoundFunctions<typeof
   }, { timeout: 3000 })
 }
 
+export const checkStatTooltipTotalCarbs = async (patientStatistics: BoundFunctions<typeof queries>, infoIconLabel: string, expectedTextContent: string) => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏This function is never called